README.md

```diff
@@ -4,47 +4,81 @@ A Reveal.js theme for nimib.
 
 An example can be found in `docs/` folder and the generated slides can be seen here: https://hugogranstrom.com/nimib-reveal/
 
+# Table of contents
+- [API](#api)
+  - [Animate Code](#animate-code)
+  - [Fragments (animations)](#fragments-animations)
+    - [End animations](#end-animations)
+    - [List of fragments](#list-of-fragment-animations)
+  - [Big Text](#big-text)
+  - [Hiding code output](#hiding-code-output)
+  - [Themes](#themes)
+- [Roadmap](#roadmap-🗺)
+
 # API
+These lines are needed at the top of your nim files:
+```nim
+import nimib
+import nimiSlides
+
+nbInit(theme = revealTheme)
+initReveal()
```

Comment:
you can remove also this `initReveal` right?

Reply:
Poof! Gone! 🪄